More general types for frozenset methods

dmoisset opened the pull request:
Fix for (#276)

gvanrossum commented:
Can you fix the conflict and also do the same for 2.7? (I was going to merge but I can't fix the conflict for you.)

gvanrossum commented:
(I think the conflict is caused by adc6bf5f, which makes all these operations take `*s` instead of just `s` -- they are all varargs.)

dmoisset commented:
I've rebased my changes and updated the changes (to include varargs from adc6bf5, cover python 2.7, and update the set() methods that had the same problem that frozenset() had)

gvanrossum commented:
Thanks! When you pushed your rebase, GitHub sent me an email notifying me that you push 95 commits... That nearly gave me a heart attack. :-)

dmoisset commented:
I'm sorry, I actually pushed a wrong merge that included a lot of old
commits, so I had to clean it up :-)

On Mon, Jul 18, 2016 at 9:18 PM, Guido van Rossum <notifications@github.com>
wrote:

> Thanks! When you pushed your rebase, GitHub sent me an email notifying me
> that you push 95 commits... That nearly gave me a heart attack. :-)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/python/typeshed/pull/277#issuecomment-233496978>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAMQHTG9uLQrmwPDV2LEApH6zk4h67tuks5qXBfvgaJpZM4IxGeQ>
> .
>



-- 
Daniel F. Moisset - UK Country Manager
www.machinalis.com
Skype: @dmoisset
